### PR TITLE
Mapper creation widget

### PR DESCRIPTION
**Pull request by EnoF**
created a mapper creation widget, the new widget is available on `/#/mapper`
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/EnoF/con-rest/pull/46%23issuecomment-69739098%22%2C%20%22https%3A//github.com/EnoF/con-rest/pull/46%23issuecomment-70052081%22%2C%20%22https%3A//github.com/EnoF/con-rest/pull/46%23issuecomment-70092800%22%2C%20%22https%3A//github.com/EnoF/con-rest/pull/46%23discussion_r22944484%22%2C%20%22https%3A//github.com/EnoF/con-rest/pull/46%23discussion_r22944656%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/EnoF/con-rest/pull/46%23issuecomment-69739098%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22just%20one%20thing%3A%20should%20we%20call%20it%20maps%20or%20map%20in%20the%20db%3F%22%2C%20%22created_at%22%3A%20%222015-01-13T12%3A46%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/243319%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dominikkukacka%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40TheBay0r%20You%20are%20right%2C%20%3E_%3C%20major%20fail%22%2C%20%22created_at%22%3A%20%222015-01-15T08%3A07%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1508400%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/EnoF%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-01-15T14%3A30%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8627051%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TheBay0r%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20ded6335aa5ea3b90bf6a22650bbe78b73aff9eb4%20app/viewModels/mapperC.js%2024%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/EnoF/con-rest/pull/46%23discussion_r22944656%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Couldn%27t%20find%20were%20this%20function%20is%20called%20%22%2C%20%22created_at%22%3A%20%222015-01-14T15%3A44%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8627051%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TheBay0r%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20app/viewModels/mapperC.js%3AL1-33%22%7D%2C%20%22Pull%20ded6335aa5ea3b90bf6a22650bbe78b73aff9eb4%20app/widgets/mapperRegistrator/mapperRegistrator.html%2025%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/EnoF/con-rest/pull/46%23discussion_r22944484%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Shouldn%27t%20there%20be%20another%20Button%20to%20actually%20save%20the%20maps%3F%22%2C%20%22created_at%22%3A%20%222015-01-14T15%3A43%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8627051%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TheBay0r%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20app/widgets/mapperRegistrator/mapperRegistrator.html%3AL1-29%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/TheBay0r%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8627051%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/TheBay0r'></a>

- [ ] <a href='#crh-comment-Pull ded6335aa5ea3b90bf6a22650bbe78b73aff9eb4 app/widgets/mapperRegistrator/mapperRegistrator.html 25'></a> &nbsp;<b><a href='https://github.com/EnoF/con-rest/pull/46#discussion_r22944484'>File: app/widgets/mapperRegistrator/mapperRegistrator.html:L1-29</a></b>
- <a href='https://github.com/TheBay0r'></a> Shouldn't there be another Button to actually save the maps?
- [ ] <a href='#crh-comment-Pull ded6335aa5ea3b90bf6a22650bbe78b73aff9eb4 app/viewModels/mapperC.js 24'></a> &nbsp;<b><a href='https://github.com/EnoF/con-rest/pull/46#discussion_r22944656'>File: app/viewModels/mapperC.js:L1-33</a></b>
- <a href='https://github.com/TheBay0r'></a> Couldn't find were this function is called
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/EnoF/con-rest/pull/46#issuecomment-69739098'>General Comment</a></b>
- <a href='https://github.com/dominikkukacka'></a> just one thing: should we call it maps or map in the db?
- <a href='https://github.com/EnoF'></a> @TheBay0r You are right, >_< major fail

<a href='https://www.codereviewhub.com/EnoF/con-rest/pull/46?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/EnoF/con-rest/pull/46?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/EnoF/con-rest/pull/46?approve=0'></a>&nbsp;<a href='https://github.com/EnoF/con-rest/pull/46'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by EnoF** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/EnoF/con-rest/pull/46'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by dominikkukacka** [comment]
just one thing: should we call it maps or map in the db?

File: app/widgets/mapperRegistrator/mapperRegistrator.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            <md-text-float label="source"
+                           ng-model="map.source"></md-text-float>
+            <md-text-float label="destination"
+                           ng-model="map.destination"></md-text-float>
+            <md-button class="md-fab"
+                       aria-label="remove map"
+                       ng-disabled="$first && $last"
+                       ng-click="removeMap($index)">
+                <md-icon icon="/icons/ic_delete_24px.svg"></md-icon>
+            </md-button>
```
Comment (TheBay0r) [inline app/widgets/mapperRegistrator/mapperRegistrator.html]: Shouldn't there be another Button to actually save the maps?

File: app/viewModels/mapperC.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        };
+
+        $scope.removeMap = function removeMap(index) {
+            if ($scope.maps.length === 1) {
+                return;
+            }
+            $scope.maps.splice(index, 1);
+        };
+
+        $scope.save = function save() {
```
Comment (TheBay0r) [inline app/viewModels/mapperC.js]: Couldn't find were this function is called

**Comment by EnoF** [comment]
@TheBay0r You are right, >_< major fail

**Comment by TheBay0r** [comment]
:shipit:

**Pull request merged**